runtime/kusama/src/lib.rs

```diff
@@ -132,6 +133,13 @@ parameter_types! {
 	pub const Version: RuntimeVersion = VERSION;
 }
 
+parameter_types! {
+	pub const DbWeight: RuntimeDbWeight = RuntimeDbWeight {
```

Comment:
How is this determined?

Reply:
There is a trie benchmark in substrate

`cargo run --release -p node-bench -- trie` 

Those are a values given there are 200k keys in the database

Reply:
Ok, seems arbitrary, but better than nothing

Reply:
> Ok, seems arbitrary, but better than nothing

It can be updated with governance if database grows :)